required/latex-lab/latex-lab-toc-kernel-changes.dtx

```diff
@@ -151,6 +151,28 @@
 \def\toclevel@subparagraph{5}
 \def\toclevel@figure{1}
 \def\toclevel@table{1}
+%    \end{macrocode}
+%\subsection{Storing the relation between target names and structure numbers}
+% To be able to add a /Ref key to structures the tagging makes use of target names
+% and stores the relationship in a property. We add a hook to \cs{MakeLinkTarget} to 
```

Comment:
I really prefer a few  `%` lines (all over the place not just here) to make it easier to see where sections start in the source.